src/test/java/com/google/javascript/cl2dts/multi_class.d.ts

```diff
@@ -10,6 +10,8 @@ declare namespace ಠ_ಠ.cl2dts_internal.multi_class {
   }
   interface I2 extends I {
   }
+  class C extends B {
+  }
```

Comment:
can you add `class D implements I` for completeness of all possible cases.
- class implements
- class extends
- class implements and extends
- interface extends